cob_hardware_config/cob3-2/config/joint_limits.yaml

```diff
@@ -0,0 +1,99 @@
+# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
```

Comment:
I assume this has been copied from cob_moveit_config. So we don't need to review in detail. @ipa-nhg: right?